MPCAutofill/cardpicker/templates/cardpicker/credits.html

```diff
@@ -31,15 +31,15 @@ <h3>Contribution Guidelines</h3>
         <thead>
         <tr>
             <th scope="col">ID</th>
-            <th scope="col" style="text-align: center">Drive</th>
+            <th scope="col">Drive</th>

```

Comment:
prefer keeping this header centred as the row elements are centred:

![image](https://user-images.githubusercontent.com/3079166/177117320-d51001ea-9d6a-47fc-8504-0209af1ae749.png)

Reply:
Fixed! Have no idea how that happened lol